lib/async_hooks.js

```diff
@@ -45,6 +50,7 @@ const {
   disableHooks,
   updatePromiseHookMode,
   executionAsyncResource,
+  topLevelResource,
```

Comment:
why do you need the topLevelResource?

Reply:
There is no `after` hook emitted for the top-level resource. So when an `AsyncLocalStorageDisposableStore` is created in the top-level resource scope, a next tick callback is scheduled to perform the leakage check instead.